src/features/todos.ts

```diff
@@ -1,9 +1,26 @@
 import { Todo } from '../types/Todo';
 
-export const actions = {};
+type SetTodos = {
+  type: 'todos/SET';
+  payload: Todo[];
+};
+
+const setTodos = (todos: Todo[]): SetTodos => ({
+  type: 'todos/SET',
+  payload: todos,
+});
+
+export const actions = { setTodos };
+type State = Todo[] | [];
```

Comment:
```suggestion
type State = Todo[];
```
it's acceptable with arrays